CONTRIBUTING.md

```diff
@@ -1,55 +1,156 @@
-# Contributing to Real-time Ubuntu docs
+# Contributing to Real-time Ubuntu documentation
 
-The [Real-time Ubuntu documentation] lives in the `docs/` directory and is deployed on [Read the Docs].
-It is written in [reStructuredText] and built with [Sphinx].
+We believe that everyone has something valuable to contribute, whether you're a
+coder, a writer, or a tester.
 
-The documentation should follow the [Canonical documentation style guide] in use for documentation at Canonical.
+The guidelines below will help keep your contributions effective and meaningful.
 
-## Fork and download the docs  
+## Code of conduct
 
-If you are working with this documentation set for the first time, you'll need to create a fork of this repository on your GitHub account, and clone the forked repository.
+When contributing, you must abide by the [Ubuntu Code of Conduct].
 
-## Install dependencies
+## Licence and copyright
 
-Go to the `docs/` directory and run:
+Unless explicitly stated in the license headers of source files, all
```

Comment:
This is the part where we indicate that certain files will be licensed differently if there are explicit license headers.
We can send this part to the Legal team for review to ensure the wording is accurate.